db/db.go

```diff
@@ -95,14 +96,63 @@ func (d *DbClient) AddMemberToSession(ctx context.Context, refId string, newMemb
 		return nil, fmt.Errorf("failed to map document data: %w", err)
 	}
 
-	// Update member session
-	session.MembersSession = append(session.MembersSession, newMember)
-	_, err = docRef.Set(ctx, map[string]interface{}{
-		"members_sessions": session.MembersSession,
-	}, firestore.MergeAll)
+	// Get members, check if newMember already joined
+	members, _ := d.GetMembersList(ctx, refId)
+
+	if slices.Contains(members.MembersSession, newMember) {
```

Comment:
this is so sad, we combine business logic with db connection. we need to seperate this. but later

Reply:
can we put the logic inside handler fow now?